CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.3.1
+February 14, 2018
+
+### Bug Fixes
+* Change 'murmurhash' to 'murmurhash-signed' in OptimizelySDK.nuspec .
```

Comment:
I would actually mention the exact problem.  Like potential missing dll or dll mismatch.